### ISSUE TITLE
🐛 Remove "Unknown" From 'is_primary_treatment"

### ISSUE THREAD

**Issue by hknahal** [icgc-argo/argo-dictionary#356]
## Describe the bug
The current validation script for "is_primary_treatment" allows the submission of "Unknown" only if "treatment_type" = "No treatment". However, since "is_primary_treatment" is a conditional field, the users can just leave it empty when there is no treatment. The permissible value `Unknown` is not needed anymore.

## Implementation
- Remove "Unknown" from value list. It only accepts Yes or No or empty.
- Update the validation script to take this into account (only allow "is_primary_treatment" = "" if "treatment_type" = "no treatment")
- If "Unknown" is required, then a clinical exception will be required.

Example:
https://platform-ui.qa.argo.cancercollaboratory.org/submission/program/ALEXIS-INTL/clinical-submission?tab=treatment
In this submission, 
current behaviour: 
treatment_type = Surgery, is_primary_treatment = Unknown
Expected:
treatment_type = Surgery, is_primary_treatment = Not applicable
![Screen Shot 2022-08-24 at 11 24 15 AM](https://user-images.githubusercontent.com/8744047/186462379-ba00d302-cfde-422b-a89b-c048f7d072cf.png)

To summarize the correct scenarios:
|   | treatment_type                            | is_primary_treatment |              Valid?               |    |
|---|-------------------------------------------|----------------------|-----------------------------|---|
| 1 | 'No treatment'                              | ''       |            Valid                 |   |
| 2 | anything but 'No treatment', e.g. 'Surgery' | 'Yes' or 'No'            |                Valid             |   |
| 3 |                                           | Unknown              | requires clinical exception |   |

**Comment by Buwujiu** [comment]
Tested in QA TEST-CA, working as expected!

**Comment by hknahal** [comment]
Released in Dictionary version 1.16
